torch/_dynamo/metrics_context.py

```diff
@@ -109,15 +109,16 @@ def set_key_value(self, metric: str, key: str, value: Any) -> None:
             self._metrics[metric] = {}
         self._metrics[metric][key] = value
 
-    def update(self, values: dict[str, Any]) -> None:
```

Comment:
Nit - Can we add a test explicitly for overwrite on metrics_context(seperate from the graph test).